modules/ossm-rn-deprecated-features.adoc

```diff
@@ -15,6 +15,25 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 Removed functionality no longer exists in the product.
 
+== Deprecated and removed features {SMProductName} 2.3.1
+
+The `concurrency` field of the `spec.proxy` API is deprecated as of release 2.3.1, and is planned for removal in a future release. Although the field is still supported in release 2.3.1, customers should begin using the `concurrency` field of the `ProxyConfig` API.
+
+The `concurrency` field specifies the number of worker threads to run. If the field is set to `0`, then the number of worker threads available is equal to the number of CPU cores. If the field is not set, then the number of worker threads available defaults to `2`.
+
+In the following example, the `concurrency` field is set to `0`.
```

Comment:
Instead of explaining what this is, I would say that the following ProxyConfig is the equivalent of `spec.proxy.concurrency` in the SMCP.

Reply:
@jewertow As a documentation practice, it's good to explain how to use something, so I'd like to keep this information. Some customers who want to do this might not already be familiar with spec.proxy.concurrency so saying they work the same might not help customers just learning to do this.

I also think you're bringing up a point I hadn't thought of - they basically work the same. So, maybe saying that provides value for customers already familiar with spec.proxy.concurrency.

I'll add a sentence saying they are equivalent prior to the explanation.